node-mixin/dashboards/node.libsonnet

```diff
@@ -0,0 +1,176 @@
+          span=6,
+          format='percent',
+          max=100,
+          min=0,
+        )
+        .addTarget(prometheus.target(
+          |||
+            100 - (avg by (cpu) (irate(node_cpu{%(nodeExporterSelector)s, mode="idle", instance="$instance"}[5m])) * 100)
+          ||| % $._config,
+          legendFormat='{{cpu}}',
+          intervalFactor=10,
+        ));
+
+      local systemLoad =
+        graphPanel.new(
+          'System load',
+          datasource='$datasource',
+          span=6,
+          format='percent',
+        )
+        .addTarget(prometheus.target('node_load1{%(nodeExporterSelector)s, instance="$instance"} * 100' % $._config, legendFormat='load 1m'))
+        .addTarget(prometheus.target('node_load5{%(nodeExporterSelector)s, instance="$instance"} * 100' % $._config, legendFormat='load 5m'))
+        .addTarget(prometheus.target('node_load15{%(nodeExporterSelector)s, instance="$instance"} * 100' % $._config, legendFormat='load 15m'));
+
+      local memoryGraph =
+        graphPanel.new(
+          'Memory Usage',
+          datasource='$datasource',
+          span=9,
+          format='bytes',
+        )
+        .addTarget(prometheus.target(
+          |||
+            node_memory_MemTotal{%(nodeExporterSelector)s, instance="$instance"}
+            - node_memory_MemFree{%(nodeExporterSelector)s, instance="$instance"}
+            - node_memory_Buffers{%(nodeExporterSelector)s, instance="$instance"}
+            - node_memory_Cached{%(nodeExporterSelector)s, instance="$instance"}
+          ||| % $._config, legendFormat='memory used'
+        ))
+        .addTarget(prometheus.target('node_memory_Buffers{%(nodeExporterSelector)s, instance="$instance"}' % $._config, legendFormat='memory buffers'))
+        .addTarget(prometheus.target('node_memory_Cached{%(nodeExporterSelector)s, instance="$instance"}' % $._config, legendFormat='memory cached'))
+        .addTarget(prometheus.target('node_memory_MemFree{%(nodeExporterSelector)s, instance="$instance"}' % $._config, legendFormat='memory free'));
+
+      local memoryGauge = gauge.new(
+        'Memory Usage',
+        |||
+          (
+            node_memory_MemTotal{%(nodeExporterSelector)s, instance="$instance"}
+          - node_memory_MemFree{%(nodeExporterSelector)s, instance="$instance"}
+          - node_memory_Buffers{%(nodeExporterSelector)s, instance="$instance"}
+          - node_memory_Cached{%(nodeExporterSelector)s, instance="$instance"}
+          ) * 100
+            /
+          node_memory_MemTotal{%(nodeExporterSelector)s, instance="$instance"}
+        ||| % $._config,
+      ).withLowerBeingBetter();
+
+      local diskIO =
+        graphPanel.new(
+          'Disk I/O',
+          datasource='$datasource',
+          span=9,
+        )
+        .addTarget(prometheus.target('sum by (instance) (rate(node_disk_bytes_read{%(nodeExporterSelector)s, instance="$instance"}[2m]))' % $._config, legendFormat='read'))
```

Comment:
These are the old names

Reply:
`node_disk_bytes_read` you mean?  whats the new name?

Reply:
node_disk_bytes_read_total. There's a few others too.

Reply:
I see; they've changes in master but are in a (non RC) release yet.  This is going to be pretty horrible for users...